models/articles.models.js

```diff
@@ -27,4 +27,17 @@ function fetchArticles() {
     });
 }
 
-module.exports = { fetchArticles, fetchArticleById };
+function updateArticleById(articleId, incVotes) {
+  return db
+    .query(
+      `UPDATE articles SET votes=$1+votes WHERE article_id=$2 RETURNING *`,
+      [incVotes, articleId]
+    )
+    .then(({ rows }) => {
+      if (!rows.length)
```

Comment:
I know this works, but for readability don't forget your { } this isn't Python 😄